### ISSUE TITLE
Mode 10: CP crashes

### ISSUE THREAD

**Issue by AndyK70** [Courseplay/courseplay#2434]
First unload of grass backwards into the silo 34.000l went good, btw.
The CaseIH Puma had the shield at the back, no other tool was attached.
First pass was doing good. cp drove back (forward) to the beginning of the silo and then the issue happened. 
I wasn't able to stop cp, switch vehicles, esc to menu or any other action except for one: I could open the console and type in 'quit' ...  :cry:

[FS17 log.txt](https://gist.github.com/AndyK70/aec37d0000c6821eba34b17837eb4320#file-fs17-log-txt)

Mode 10 settings: 
![grafik](https://user-images.githubusercontent.com/666825/32420450-be74a690-c28a-11e7-9268-c4e07ad89e22.png)

![farmingsimulator2017game_2017_11_06_00_15_24_921_1](https://user-images.githubusercontent.com/666825/32420411-31634cca-c28a-11e7-9413-44c998411c7f.jpg)

PS: was unsure if to post in the test-thread #1568 or make a new one, sry if I picked the wrong choice 😳 

PPS: did another test, with console open to see, when exactly the error occurs and as soon as I started cp (Case IH Puma with the silage shield) the errors showed up. I thought you may wanted to know.
All stock vehicles and attachments used on stock Goldcrest Valley.

**Comment by pops64** [comment]
Could we get a copy of your savegame with how you were able to get this happen has well has your courses

**Comment by AndyK70** [comment]
here you go, everything included, even the mods I use: [FarmingSimulator2017.zip](https://www.dropbox.com/s/gb50wa7e3j8e9n8/FarmingSimulator2017.zip?dl=0)

extract the zipfile in the folder "My Games\FarmingSimulator2017\" and you are ready to go with savegame4.
Just check the settings seen below in the screenshots and start the Puma.

![fsscreen_2017_11_11_21_15_00_1](https://user-images.githubusercontent.com/666825/32693166-6496d62a-c726-11e7-98a2-c92346c7357f.jpg)
![fsscreen_2017_11_11_21_15_19_1](https://user-images.githubusercontent.com/666825/32693167-64b28528-c726-11e7-87b1-9164bc5675d4.jpg)

Interesting fact though: There was no error, before the Fendt 939 did unload the grass into the silo. I set the course, started the Case IH Puma and it went through the silo without errors. After the Fendt unloaded the grass and I started the Case IH Puma again (without any modification) the error appears instantaneously.

**Comment by pops64** [comment]
Problem still persisting?

**Comment by AndyK70** [comment]
I didn't dare trying it again with my grown savegame. For the time being I always was compacting and leveling by myself.
I'll set up a new savegame with CP as only mod and try it out again.
Hopefully I can narrow down this issue.

**Comment by AndyK70** [comment]
I did some testing, without mods (except CP), with some mods, with all mods I usually use and didn't found that issue.
So I made a backup of my actual savegame which is the same where this issue happened and started mode10 with 'leveling' then with 'buildup' each in combination with tipping 'drive through' and 'backwards' and that error didn't show up anymore.
I'm a bit puzzled, but happy. 😄 

So I think this issue can be closed.
